Makefile

```diff
@@ -17,6 +17,7 @@ dependencies:
 	go get -u github.com/NebulousLabs/errors
 	go get -u github.com/NebulousLabs/go-upnp
 	go get -u github.com/NebulousLabs/muxado
+	go get -u github.com/NebulousLabs/threadgroup
```

Comment:
renter now uses the NL threadgroup instead of the Sia threadgroup. The NL threadgroup has a few changes:

+ OnStop and AfterStop functions return errors
+ No Flush()
+ Safe to double-add e.g. `tg.Add(); tg.Add(); tg.Done(); tg.Done()` is safe in the NL threadgroup but no the Sia threadgroup.

Reply:
should we be gradually phasing out `sync/threadgroup` elsewhere in the code? Seems worth doing